<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestComplex v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="workSpace"/>
      <widget class="QSplitter" name="horizontalSplitter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTextBrowser" name="textBrowser"/>
       <widget class="QTextBrowser" name="logSpace">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::WidgetWidth</enum>
        </property>
        <property name="lineWrapColumnOrWidth">
         <number>0</number>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt; font-weight:96;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="CommandLine" name="commandLine">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_workspace"/>
    <addaction name="actionOpen_workspace"/>
    <addaction name="actionSave_workspace"/>
    <addaction name="actionSave_workspace_as"/>
    <addaction name="actionDelete_workspace"/>
    <addaction name="separator"/>
    <addaction name="actionNew_test"/>
    <addaction name="actionOpen_test"/>
    <addaction name="actionSave_test"/>
    <addaction name="actionSave_test_as"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_to_test"/>
    <addaction name="actionAutoDetect"/>
    <addaction name="actionAdd_existing_file"/>
    <addaction name="actionRemove_from_test"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionReport"/>
    <addaction name="actionClean_logspace"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionMaketest1"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTests"/>
    <addaction name="actionLogs"/>
    <addaction name="actionReports"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfigure_serial_port"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMy_test_buttons">
    <property name="title">
     <string>My test buttons</string>
    </property>
    <addaction name="actionRead_log"/>
    <addaction name="actionMake_SNR_test"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
   <addaction name="menuMy_test_buttons"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_workspace">
   <property name="text">
    <string>New workspace</string>
   </property>
  </action>
  <action name="actionOpen_workspace">
   <property name="text">
    <string>Open workspace</string>
   </property>
  </action>
  <action name="actionSave_workspace">
   <property name="text">
    <string>Save workspace</string>
   </property>
  </action>
  <action name="actionSave_workspace_as">
   <property name="text">
    <string>Save workspace as...</string>
   </property>
  </action>
  <action name="actionNew_test">
   <property name="text">
    <string>New test</string>
   </property>
  </action>
  <action name="actionOpen_test">
   <property name="text">
    <string>Open test</string>
   </property>
  </action>
  <action name="actionSave_test">
   <property name="text">
    <string>Save test</string>
   </property>
  </action>
  <action name="actionSave_test_as">
   <property name="text">
    <string>Save test as...</string>
   </property>
  </action>
  <action name="actionAdd_to_test">
   <property name="text">
    <string>Add to test</string>
   </property>
  </action>
  <action name="actionAutoDetect">
   <property name="text">
    <string>AutoDetect</string>
   </property>
  </action>
  <action name="actionAdd_existing_file">
   <property name="text">
    <string>Add existing log file</string>
   </property>
  </action>
  <action name="actionRemove_from_test">
   <property name="text">
    <string>Remove from test</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionTests">
   <property name="text">
    <string>Tests</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionConfigure_serial_port">
   <property name="text">
    <string>Configure serial port...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete_workspace">
   <property name="text">
    <string>Delete workspace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRead_log">
   <property name="text">
    <string>Read log file</string>
   </property>
  </action>
  <action name="actionMake_SNR_test">
   <property name="text">
    <string>Make SNR test</string>
   </property>
  </action>
  <action name="actionClean_logspace">
   <property name="text">
    <string>Clean logspace</string>
   </property>
  </action>
  <action name="actionMake_some_noize">
   <property name="text">
    <string>Make some noize</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionMaketest1">
   <property name="text">
    <string>Make test 1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CommandLine</class>
   <extends>QLineEdit</extends>
   <header>commandLine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
